src/app/Domain/state/cloth/cloth.reducer.ts

```diff
@@ -29,9 +30,10 @@ const _clothReducer = createReducer(
             size: currentValue[9],
             price: currentValue[11],
             status: currentValue[13],
+            photo: currentValue[15],
           });
-          return accumulator;
-        }, []).slice(1);
+          return accumulator.filter(accumulator => accumulator.status == "Disponible");
```

Comment:
```suggestion
          return accumulator.filter(accumulator => accumulator.status == this.value);
```
You can use here the variable. Nevertheless, the name `value` doesn't really makes any sense to what it actually does.

Reply:
In the line 34 the slice index would be 0 if not is excluding the first element.